apis/appmesh/v1beta2/virtualrouter_types.go

```diff
@@ -39,6 +39,10 @@ type WeightedTarget struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=100
 	Weight int64 `json:"weight"`
+	// Specifies the targeted port of the weighted object
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	Port *int64 `json:"port,omitempty"`
 }
 
```

Comment:
Do we need to update GrpcRouteMatch, TcpRouteMatch, and TCPRoute as well?

Reply:
is tcp route match and tcp route new? could you link me the spec for that? cc @rakeb

Reply:
actually i found it here https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_TcpRouteMatch.html